mitxpro/views.py

```diff
@@ -41,6 +41,20 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
     return render(request, "index.html", context=get_js_settings_context(request))
 
 
+def handler404(request, exception):
+    """404: NOT FOUND ERROR handler"""
+    return render(
+        request, "404.html", context=get_js_settings_context(request), status=404
+    )
+
+
+def handler500(request):
+    """500 INTERNAL SERVER ERROR handler"""
+    return render(
+        request, "500.html", context=get_js_settings_context(request), status=500
+    )
```

Comment:
Coverage is decreasing, please add couple of tests for these views.

Reply:
I'll add a test for the 404 handler, for 500 server errors it is advised to use integration testing, not unit testing and the test client provides no way to deal with it without adding a url that intentionally raises an exception which I doubt we would want to do.